### DISCUSSION TITLE
private_key as string

### DISCUSSION THREAD

**Discussion by GinJus** [simonschaufi/laravel-dkim#27] [Ideas]
Is it an idea to provide an option to use the private_key as a string? I have multiple domains in my project and would like to use the Config::set function to dynamically change the key.

**Comment by simonschaufi** [comment]
Sure, I could do that. Would you also sponsor this feature?

**Reply by GinJus** [reply]
Depends on the cost. It is a personal project, so I have to pay for it myself.
